integration-tests/tests/src/tests/alias.ts

```diff
@@ -0,0 +1,151 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { expect } from "chai";
+import Realm from "realm";
+import { openRealmBeforeEach } from "../hooks";
+
+type ObjectA = {
+  otherName: string;
+  age?: number;
+};
+
+function addTestObjects(realm: Realm) {
+  realm.write(() => {
+    realm.create("ObjectA", {
+      otherName: "Foo",
+      age: 41,
+    });
+    realm.create("ObjectA", {
+      otherName: "Bar",
+      age: 42,
+    });
+  });
+}
+
+const PersonSchema: Realm.ObjectSchema = {
+  name: "Person",
+  properties: {
+    _name: { type: "string", mapTo: "name" },
+    address: { type: "string", indexed: true },
+    age: "double",
+    _married: { type: "bool", default: false, mapTo: "married" },
+    _children: { type: "list", objectType: "Person", mapTo: "children" },
+    _parents: { type: "linkingObjects", objectType: "Person", property: "children", mapTo: "parents" },
+  },
+};
+
+const ObjectASchema: Realm.ObjectSchema = {
+  name: "ObjectA",
+  properties: {
+    otherName: { type: "string", mapTo: "name" },
+    age: { type: "int", optional: true },
+  },
+};
+
+describe("Aliasing property names using mapTo", () => {
+  openRealmBeforeEach({
+    schema: [PersonSchema, ObjectASchema],
+  });
+
+  afterEach(() => {
+    Realm.clearTestState();
+  });
```

Comment:
I see that this is not clear from the (lack of) documentation of the `openRealmBeforeEach` hook, this actually automatically registered by the `openRealmBeforeEach` hook. I need to document that ...

```suggestion
```